.github/workflows/build-test-measure.yml

```
@@ -103,18 +103,24 @@ jobs:
         run: npm run lint:pkg-json
 
       - name: Detect ESLint coding standard violations
-        if: github.event.pull_request.head.repo.fork == true
+        if: >
+          github.event.pull_request.head.repo.fork == true ||
+          github.event.pull_request.user.login == 'dependabot'
         run: npm run lint:js
 
       - name: Generate ESLint coding standard violations report
-        # Prevent generating the ESLint report if run from a PR fork
-        if: github.event.pull_request.head.repo.fork == false
+        # Prevent generating the ESLint report if PR is from a fork or authored by Dependabot.
+        if: >
+          github.event.pull_request.head.repo.fork == false &&
+          github.event.pull_request.user.login != 'dependabot'
```

Since this needs to be strictly the inverse of the previous condition, maybe it would be easier to maintain if it was structured like:

```suggestion
          ! ( github.event.pull_request.head.repo.fork == true ||
          github.event.pull_request.user.login == 'dependabot' )
```

Same for other instances below.

---

Done in https://github.com/ampproject/amp-wp/pull/6253/commits/9549afcea0fb376890eb4932b3e2e8ace75c0e3a.